File: app/public_html/script/functions/samples/30.py
x = 0, y = 0
def loop():
    global x, y
    if (getX() == x and getY() == y) or not isSafeThere(x,y):
        x = randint(0,250)/10
        y = randint(0,250)/10
     
    if isSafeThere(x,y):
        if getAp() > 70 and getDist(x,y) > 2:
            teleport(x,y)
        else:
            moveTo(x,y)
     
 

File: app/public_html/script/functions/samples/30.c
float x,y;
loop(){
    
    if ((getX() == x && getY() == y) || !isSafeThere(x,y)){
        x = (rand()%250)/10;
        y = (rand()%250)/10;
    }
    
    if(isSafeThere(x,y)){
        if (getAp() > 70 && getDist(x,y) > 2)
            teleport(x,y);
        else
            moveTo(x,y);
    }
}